### PR TITLE
Improve readme and rename ha-availability-set, ha-availability-zones directories 

### PR DESCRIPTION
**Pull request by chkp-romanka**

### CODE REVIEW

File: terraform/azure/ha-availability-set-new-vnet/README.md [hunk 1]
```diff
@@ -10,19 +10,41 @@ As part of the deployment the following resources are created:
...
+- /terraform/azure/modules/common - used for creating a resource group and defining common variables.
+- /terraform/azure/modules/vnet - used for creating new virtual network and subnets.
+- /terraform/azure/modules/network-security-group - used for creating new network security groups and rules.
 
 
 ## Configurations
 - Install and configure Terraform to provision Azure resources: [Configure Terraform for Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/terraform-install-configure)
-- In order to use ssh connection to VMs, add a public key to the /cgi-terraform/azure/ha-availability-set/azure_public_key file
+- In order to use ssh connection to VMs, it is **required** to add a public key to the /terraform/azure/ha-availability-set/azure_public_key file
+<br>In case there is no need in the ssh key usage, next lines in the main.tf file need to be deleted or commented:
```
Comment (chkp-dmitrygo) [inline terraform/azure/ha-availability-set-new-vnet/README.md]: "**the** next lines"

File: terraform/azure/ha-availability-set-new-vnet/README.md [hunk 2]
```diff
@@ -10,19 +10,41 @@ As part of the deployment the following resources are created:
...
 
+        ssh_keys {
```
Comment (chkp-dmitrygo) [inline terraform/azure/ha-availability-set-new-vnet/README.md]: can this be in code format?

File: terraform/azure/ha-availability-set-new-vnet/README.md [hunk 3]
```diff
@@ -10,19 +10,41 @@ As part of the deployment the following resources are created:
...
+          path = "/home/notused/.ssh/authorized_keys"
+          key_data = file("${path.module}/azure_public_key")
+        }
 ## Usage
-- Create a Service Principal with the following permissions to the Azure subscription. 
-This service principal will be used by Terraform in order to deploy the solution.
-- Fill all variables in the /cgi-terraform/azure/ha-availability-set/terraform.tfvars file with proper values (see below for variables descriptions).
+- Choose the preferred login method to the Azure in order to deploy the solution:
```
Comment (chkp-dmitrygo) [inline terraform/azure/ha-availability-set-new-vnet/README.md]: "method to Azure"

File: terraform/azure/ha-availability-set-new-vnet/README.md [hunk 4]
```diff
@@ -10,19 +10,41 @@ As part of the deployment the following resources are created:
...
+    - Create a [Service Principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal) (or use the existing one) 
+    - Grant the Service Principal at least "**Owner**" permissions to the Azure subscription<br>
+
+    <br>2. Using **az** commands from a command-line:
+    - Run  **az login** command 
+    - Sign in with your account credentials in the browser
+    - [Accept Azure Marketplace image terms](https://docs.microsoft.com/en-us/cli/azure/vm/image/terms?view=azure-cli-latest) by running:
+     <br>**az vm image terms accept --urn publisher:offer:sku:version**, where:
+        - publisher = checkpoint;
+        - offer = vm_os_offer(see accepted values in the table below);
```
Comment (chkp-dmitrygo) [inline terraform/azure/ha-availability-set-new-vnet/README.md]: add a space before the bracket - _offer (see

File: terraform/azure/ha-availability-set-new-vnet/README.md [hunk 5]
```diff
@@ -98,6 +120,14 @@ This service principal will be used by Terraform in order to deploy the solution
     bootstrap_script                = "touch /home/admin/bootstrap.txt; echo 'hello_world' > /home/admin/bootstrap.txt"
     allow_upload_download           = true
     disable_password_authentication = false
+##Revision History
+In order to get template version refer to the [sk116585](https://supportcenter.checkpoint.com/supportcenter/portal?eventSubmit_doGoviewsolutiondetails=&solutionid=sk116585)
```
Comment (chkp-dmitrygo) [inline terraform/azure/ha-availability-set-new-vnet/README.md]: In order to check the template version refer to sk...

File: terraform/azure/ha-availability-set-new-vnet/README.md (hunk 1 above)
Comment (chkp-romanka) [reply terraform/azure/ha-availability-set-new-vnet/README.md]: Done

File: terraform/azure/ha-availability-set-new-vnet/README.md (hunk 2 above)
Comment (chkp-romanka) [reply terraform/azure/ha-availability-set-new-vnet/README.md]: This is already in code format

File: terraform/azure/ha-availability-set-new-vnet/README.md (hunk 5 above)
Comment (chkp-romanka) [reply terraform/azure/ha-availability-set-new-vnet/README.md]: Done

File: terraform/azure/ha-availability-set-new-vnet/README.md (hunk 4 above)
Comment (chkp-romanka) [reply terraform/azure/ha-availability-set-new-vnet/README.md]: Done

File: terraform/azure/ha-availability-set-new-vnet/README.md (hunk 3 above)
Comment (chkp-romanka) [reply terraform/azure/ha-availability-set-new-vnet/README.md]: Done

**Pull request closed without merging**